addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/NetworkBindingConstants.java

```diff
@@ -8,11 +8,12 @@
  */
 package org.openhab.binding.network;
 
-import java.util.Collections;
 import java.util.Set;
 
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 
+import com.google.common.collect.Sets;
```

Comment:
Maybe we should just wait for Java 9 they will finally solve this. I remember I saw someone else giving the comment, so maybe I remembered incorrect, maybe @sjka or @kaikreuzer know what I might have been reading.